.ci/integration_test.sh

```diff
@@ -95,7 +95,7 @@ cat << EOF >> "${output}"
 	fmt.Printf("Get: %s\n", ret)
 }
 EOF
-
+    ${GOPATH}/goreturns -w  "${output}"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5307065)